components/apim-analytics/org.wso2.analytics.solutions.apim.analytics/widgets/APIMApiCreated/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "apim-api-created",
+    "version": "1.0.0",
+    "private": true,
+    "dependencies": {
+        "@wso2-dashboards/widget": "^1.4.0",
+        "@material-ui/core": "^3.9.0",
+        "@material-ui/icons": "^3.0.2",
+        "moment": "^2.24.0",
+        "react": "^16.7.0",
+        "axios": "^0.16.2",
+        "react-intl": "^2.8.0",
+        "rimraf": "^2.6.3"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "webpack -p",
+        "test": "react-scripts test --env=jsdom",
+        "eject": "react-scripts eject",
+        "clean": "rimraf dist",
+        "dev": "NODE_ENV=development webpack -d --config webpack.config.js --watch --progress",
```

Comment:
What is the different between `dev` and `build-dev` scripts? IMO we don't need both, hence remove one.

Applies for other `package.json`s as well.